backend/typescript/rest/camperRoutes.ts

```diff
@@ -24,11 +24,10 @@ const camperService: ICamperService = new CamperService();
 camperRouter.post("/register", createCampersDtoValidator, async (req, res) => {
   try {
     const campers = req.body as CreateCampersDTO;
-    const newCampers = await camperService.createCampers(
+    const checkoutSessionUrl = await camperService.createCampersCheckoutSession(
```

Comment:
I know I am not supposed to be commenting on the backend for this PR, but if we use this endpoint to create the checkout session, then at what point do we create the db entities? Should the checkout session creation be it's own GET endpoint `campers/checkoutSession`